pkg/app/metrics.go

```diff
@@ -28,6 +28,13 @@ var subscribeResponseReceivedCounter = prometheus.NewCounterVec(prometheus.Count
 	Help:      "Total number of received subscribe response messages",
 }, []string{"source", "subscription"})
 
+var subscribeResponseFailedCounter = prometheus.NewCounterVec(prometheus.CounterOpts{
+	Namespace: "gnmic",
+	Subsystem: "subscribe",
+	Name:      "number_of_failed_subscribe_response_messages_total",
```

Comment:
Let's call this `number_of_failed_subscribe_request_messages_total`, the error is received at the gRPC level not in a SubscribeResponse message.

Reply:
done